.goreleaser.yml

```diff
@@ -56,7 +50,8 @@ signs:
       - "${artifact}"
 release:
   extra_files:
-    - glob: 'terraform-registry-manifest.json'
-      name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
+    - glob: "terraform-registry-manifest.json"
+      name_template: "{{ .ProjectName }}_{{ .Version }}_manifest.json"
+    - glob: "sboms/gl-sbom-*.cdx.json"
```

Comment:
Shouldn't this only need the merged sbom?

Reply:
Up to you guys which one we'd like to include. I just added all of them for now.

Reply:
Made the change.